packages/PriceLockup/__tests__/PriceLockup.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import PriceLockup from '../PriceLockup'
+
+describe('PriceLockup', () => {
+  const doShallow = (props = {}) => shallow(<PriceLockup {...props} />)
+
+  it('renders', () => {
+    const priceLockup = doShallow()
+
+    expect(priceLockup).toMatchSnapshot()
+  })
+
+  it('does other things', () => {
+    const priceLockup = doShallow()
+
+    expect(priceLockup).toExist()
+  })
+
+  it('passes additional attributes to the element', () => {
```

Comment:
The price lockup should add a few tests that closely resemble the acceptance criteria. For example:

* Testing the position of the dollar sign when passing or not passing props
* Testing the size of the dollar sign, dollar value, and text when composing a small, medium, or large price lockup

Reply:
There are tests that for dollar Sign position, and as for sizes of dollar sign, dollar value, text, based on the size prop, I decided to use snapshot tests as it keeps track of the tds classes being used